src/js/components/clientMirror/ActionBar.js

```diff
@@ -60,6 +61,13 @@ class ActionBar extends React.Component {
             className="glyphicon glyphicon-refresh spinning"
           />
         </button>
+        <button className="btn" onClick={() => {this.props.onSaveClick()}}>
+          <FaFloppyO /> Save
+          <span
+            style={{ display: this.props.options.isRunning ? 'inline-block' : 'none' }}
```

Comment:
I think we need to sort out the style. This works fine on a big screen but on my laptop screen it goes over onto two lines